frontend/src/game/scenes/MainMenu.ts

```diff
@@ -1,44 +1,66 @@
-import { GameObjects, Scene } from 'phaser';
-import { EventBus } from '../EventBus';
+import { GameObjects, Scene } from "phaser";
```

Comment:
Your Prettier settings might be messed up, it's changing things against the style the rest of the team is submitting. You might want to double-check it's working correctly.